### PR TITLE
Revert "SlurmGCP. Do not add empty startup scripts"

### PR DESCRIPTION
**Pull request by harshthakkar01**
Reverts GoogleCloudPlatform/cluster-toolkit#2965

This change broke integration tests 
* DAILY-test-ml-slurm 
* DAILY-test-spack-gromacs 
* DAILY-test-slurm-gcp-v6-ssd  
* DAILY-test-slurm-gcp-v6-startup-scripts 

with following error
```
Error: Invalid for_each argument

on modules/embedded/community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/main.tf line 254, in resource "google_storage_bucket_object" "login_startup_scripts":
254: for_each = {
255: for x in var.login_startup_scripts
256: : replace(basename(x.filename), "/[^a-zA-Z0-9-_]/", "_") => x
257: }
├────────────────
│ var.login_startup_scripts is a list of object, known only after apply
```

### CODE REVIEW

File: community/modules/compute/schedmd-slurm-gcp-v6-nodeset/variables.tf
Comment (nick-stroud) [inline community/modules/compute/schedmd-slurm-gcp-v6-nodeset/variables.tf:1]: Please add description to PR as to why you are reverting the change. (broken tests and any additional context you can add).

Comment (nick-stroud) [inline community/modules/compute/schedmd-slurm-gcp-v6-nodeset/variables.tf:1]: Please wait for at least 1 of the broken tests to pass before merging.

**Review by nick-stroud — APPROVED** [review]

File: community/modules/compute/schedmd-slurm-gcp-v6-nodeset/variables.tf
Comment (harshthakkar01) [reply community/modules/compute/schedmd-slurm-gcp-v6-nodeset/variables.tf:1]: Added.

**Pull request merged**